### PR TITLE
fix renderer tests by using sinon global

### PR DESCRIPTION
**Pull request by JoviDeCroock**
Tests were failing on master IE11 due to the sinon import

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -88,6 +88,8 @@ const babelOptions = (options = {}) => {
 	};
 };
 
+sauceLabs = true;
```
Comment (developit) [inline karma.conf.js]: what's this for?

Comment (JoviDeCroock) [reply karma.conf.js]: Make sauceLabs run on these tests so I can see if the IE11  tests are going well, hence the `(remove this when verified)` in said commit

Comment (developit) [reply karma.conf.js]: ah, missed the commit. Seems like our travis setup needs to be tweaked - we should be running saucelabs for branches in the main trepo, just not forks.

**Comment by JoviDeCroock** [comment]
@marvinhagemeister do we merge it with less coverage or leave it failing for now?

**Comment by marvinhagemeister** [comment]
@JoviDeCroock I'm fine with merging this, because we'll run into the same issue sooner or later when the next version of devtools comes out. Just remove the `sauceLabs = true` flag and than it's good to go on my end :+1:

**Pull request merged**